src/columns/dateColumn.tsx

```diff
@@ -22,6 +22,7 @@ const DateComponent = React.memo<CellProps<Date | null, any>>(
         // Important to prevent any undesired "tabbing"
         tabIndex={-1}
         max="9999-12-31"
+        placeholder="dd-mm-yyyy"
```

Comment:
The 'placeholder' attribute isn't applicable to an input element of type 'date' ([mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes)).